functions/.znap.source.find

```diff
@@ -9,6 +9,7 @@ if [[ -d $reply[1] ]]; then
 elif ! [[ -r $reply[1] ]]; then
   reply=( ${reply[1]}${^~ext}(NY1-.) )
 fi
+reply=(${(u)reply})
```

Comment:
I tried to find where `.znap.source.find` is used and I couldn’t find it anywhere. I guess I forgot to remove this code at some point. 😅

Can you add a separate commit that removes it? And you can of course remove the test for it from your earlier commit.

Reply:
That's funny! I couldn't find where it was being used either, but I assumed there must be a dynamic call I missed, and it had the most gnarly glob pattern so I assumed that was what #62 had to be referencing. I'll move the test concepts from `.znap.source.find` into `.znap.source` and we should be good to go.